elk-zone/elk
# feat: add new setting to disable blur for low-performance device

resolve #2529

This PR adds a new setting to allow disabling the background blur (default is enabled) for the low-performance device.

![Screenshot from 2024-01-16 17-26-20](https://github.com/elk-zone/elk/assets/1425259/a0418309-9b88-4ac6-ae6e-2ee0150d023f)

## How to test
1. Uncheck "Enable background blur" on the settings page: https://deploy-preview-2561--elk-zone.netlify.app/settings/preferences
2. Try, for example, to mute some users to see the confirmation dialog without the background blur.
